site/_posts/2017-01-16-destroy-the-monolith.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Last Week in ManageIQ: Destroy the Monolith"
+author: durandom
+date: 2017-01-16 09:24:51 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Moin](https://en.wikipedia.org/wiki/Moin), [durandom](https://github.com/durandom/) from the providers team here, 
```

Comment:
I was about to say that in Finland we use something familiar, when I saw it mentioned in the wikipedia article :smile: